### PR TITLE
preorbital edits

### PR DESCRIPTION
**Pull request by megbalk**
This PR relates to issue #1888 to create add "preorbital process" as a synonym to "preorbital process of the pars facialis of the maxilla"; issue #2385 to add "preorbital" as a region of the face and fix the synonym in "antorbital" from "perorbital" to "periorbital"; and issue #2386 to create the term "preorbital fossa". This was done with help from @helenamachado

### CODE REVIEW

**Review by paolaroncaglia — CHANGES_REQUESTED** [review]
@megbalk 
Thank you for your patience, I was on leave and didn't see this review request until now.
The checks failed on this PR, could you try again please?
I'd also recommend to make the label 'preorbital' less ambiguous, e.g. 'preorbital region' if that's what it refers to. "preorbital" could be a relational adjective instead, or a related synonym.
Thanks.

**Comment by megbalk** [comment]
@paolaroncaglia I changed the term name to "preorbital bone region" since it is specifically about the bones of the preorbital region.

**Comment by megbalk** [comment]
Can someone help me figure out why this failed?

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -22982,6 +22982,7 @@ relationship: mutually_spatially_disjoint_with UBERON:0001630 ! muscle organ
 relationship: mutually_spatially_disjoint_with UBERON:0002385 ! muscle tissue
 relationship: only_in_taxon NCBITaxon:7742 ! Vertebrata <vertebrates>
 relationship: part_of UBERON:0001434 ! skeletal system
+relationship: RO:0000052 UBERON:0008789 ! inheres in cranial fossa
```
Comment (paolaroncaglia) [inline src/ontology/uberon-edit.obo:22985]: @megbalk 
The QC failed with this message:
ERROR: Undeclared relation: RO:0000052
RO:0000052 is 'characteristic of'...? Not sure what's going on there, but that's where you might want to look :-)
Thank you.

**Comment by paolaroncaglia** [comment]
@megbalk 

> Can someone help me figure out why this failed?

I added a comment about the QC error.
Also, more in general about the other checks, my conversation with Shawn [here](https://github.com/obophenotype/uberon/pull/2448) may help.

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (megbalk) [reply src/ontology/uberon-edit.obo:22985]: @paolaroncaglia thank you!

Comment (megbalk) [reply src/ontology/uberon-edit.obo:22985]: I think I found it in the antorbital term. I removed it to see if it solves the issue. If need be I can change it to "part of" some "cranial fossa" as it does occur in there.

**Comment by megbalk** [comment]
I'm not sure what the issue is now...

**Comment by paolaroncaglia** [comment]
@megbalk 

> I'm not sure what the issue is now...

I'm not either, but I set the qc job to re-run in case it was a simple memory issue or similar.
If it still fails, I'd have suggested to ping the uberon-editors Slack channel, but I see you're not on there. Let me know if you need me to "invite" you there. (Some people are or will be on leave, so reaching out to the group would be safer than individuals.) Or try commenting again here. Fingers crossed!

**Comment by megbalk** [comment]
it seems it's erroring on the obsolete terms...I reverted the PR so it shouldn't be in this branch.

**Comment by paolaroncaglia** [comment]
@megbalk 

> it seems it's erroring on the obsolete terms...I reverted the PR so it shouldn't be in this branch.

Yes, I see the same error about the obsolete term. Let's see if others have any advice about that.

**Review by paolaroncaglia — COMMENTED** [review]
I commented on the ticket

**Comment by megbalk** [comment]
which ticket?

**Comment by paolaroncaglia** [comment]
@megbalk 

> which ticket?

When I reviewed your PR, I added my comment
"Yes, I see the same error about the obsolete term. Let's see if others have any advice about that."
in this "Conversation" tab (what I called "the ticket", but I agree it could be confusing), rather than in the "Files changed" tab. 
I still wanted to write something in the "Files changed" tab in order to "record" that I had reviewed your PR. 

If you're still having trouble with this pull request and/or others, I'd suggest posting on the Uberon-editors Slack channel, but I can't find you there. If you'd like, I could invite you to join that channel by using your gmail address which I found on the CL/Uberon editors meeting invite.

**Comment by megbalk** [comment]
ok got help from @shawntanzk.
@paolaroncaglia hopefully this works and you can review that I've made the changes you requested.

**Comment by paolaroncaglia** [comment]
Actually, I'll merge myself because GitHub is complaining about requested changes, which you did address...

**Comment by paolaroncaglia** [comment]
@megbalk please double-check that all is well, i.e. that the changes you made have been added correctly and are in the master branch. If so, please delete the skull-issue2385 branch. Thank you.

**Comment by megbalk** [comment]
@paolaroncaglia I don't see any of the changes in uberon-edit on the master branch

**Comment by paolaroncaglia** [comment]
@megbalk not sure what's going on, so as Nicole suggested elsewhere, it may be easier and quicker to re-do these edits
BUT
I think that @matentzn and/or @shawntanzk  are about to release Uberon? If so, best to wait until the release, and pull the newly released master branch before editing. Thank you.

**Comment by shawntanzk** [comment]
The release has just been merged, but releases shouldn't affect edit files, so if its not in the edit file, it wont be in the release.

**Comment by megbalk** [comment]
I'll redo these edits.

**Comment by megbalk** [comment]
I see them now! thank you!

**Pull request merged**